code/game/machinery/doors/airlock.dm

```diff
@@ -587,46 +587,46 @@ There are 9 wires.
 	verbs -= /obj/machinery/door/airlock/proc/try_wedge_item
 	verbs += /obj/machinery/door/airlock/proc/take_out_wedged_item
 
-/obj/machinery/door/airlock/proc/try_wedge_item(mob/living/user)
+/obj/machinery/door/airlock/proc/try_wedge_item()
```

Comment:
Why is removing passing actual user fixing it? That makes no sense.

Reply:
This removed, cuz this isn't proc, it's proc-verb, and if u put in () of verb any arguments, u will see input as mob and etc.